user_manual/files/webgui/custom_groups.rst

```diff
@@ -10,8 +10,11 @@ To address that, as of ownCloud 10.0, you can now create your own groups on-the-
 Here's how to use it. 
 
 .. note::
-   	- creating or renaming a custom group using an existing display name will show an error.
-	- a config switch (checkbox) makes it possible to allow duplicates (defaults to "allow").
+	Depending on your custom groups setting, configured by the ownCloud admin, custom groups may behave differently
+	
+	- creating or renaming a custom group using an existing display name will show an error
+	- a config switch in the ownCloud's admin settings (checkbox) makes it possible to allow multiple custom groups with the same name
+	- Custom group creation can be limited to the ownCloud **group admins**
```

Comment:
You should either have all starting letters capital or lower case

Reply:
- The first and the second bullet point are the same. I suggest to merge them. "Creating or renaming a Custom Group using an existing name of another Custom Group can be allowed or not depending on administrative settings."
- Please always write Custom Groups in caps as it's a name.

Reply:
I don't know why, but when I look at the PR I don't see your last comment. Maybe because I already did a commit and you are commenting on an outdated version. Just for your info.

I will implement your requested changes though.